build_exercise_proj/CMakeLists.txt

```
@@ -1,24 +1,25 @@
-cmake_minimum_required(VERSION 3.1)
-project(build_exercise)
+cmake_minimum_required(VERSION 3.1) 
```

Increase the version to at least 3.5.
3.1 is very ancient
```suggestion
cmake_minimum_required(VERSION 3.5) 
```